bigframes/geopandas/geoseries.py

```diff
@@ -93,3 +93,29 @@ def to_wkt(self: GeoSeries) -> bigframes.series.Series:
         series = self._apply_unary_op(ops.geo_st_astext_op)
         series.name = None
         return series
+
+    def difference(self: GeoSeries, other: GeoSeries) -> GeoSeries:  # type: ignore
+        """
+        Returns a GeoSeries of the points in each aligned geometry that are not
+        in other.
+
+        The operation works on a 1-to-1 row-wise manner
+
+        Args:
+            other (GeoSeries or geometric object):
+                The GeoSeries (elementwise) or geometric object to find the
+                difference to.
+
+        Returns:
+            bigframes.geopandas.GeoSeries:
+                A GeoSeries of the points in each aligned geometry that are not
+                in other.
+
+        Raises:
+            NotImplementedError:
+                GeoSeries.difference is not supported. Use
+                bigframes.bigquery.st_difference(series), instead.
+        """
+        raise NotImplementedError(
+            f"GeoSeries.difference() is not supported. Use bigframes.bigquery.st_difference(series), instead. {constants.FEEDBACK_LINK}"
+        )
```

Comment:
Since this operation is independent of the projection, I think we can safely do this for the user. See: https://postgis.net/documentation/faq/geometry-or-geography/ for a list of methods to be cautious about.

Reply:
That is to say, let's not raise because I think the results should actually be quite close to that of geopandas. Same for any of the set-like operations that return another geometry/geography object.